packages/block-library/src/post-template/editor.scss

```diff
@@ -4,4 +4,11 @@
 		margin-left: 0;
 		list-style: none;
 	}
+
+	.wp-block-post-template {
+		.is-root-container {
```

Comment:
Is this change needed? `padding-left` and `padding-right` are already set in the rule above.

Reply:
yes, there is padding coming from somewhere, this is for WYSIWYG experience.